### ISSUE TITLE
Portable...???

### ISSUE THREAD

**Issue by Jasantes** [adeii/supermium-portable#10]
Profile in C:\Users... This is a joke!!!

**Comment by adeii** [comment]
OK. Is this version works for you? https://portableapps.com/apps/internet/supermium-portable

**Comment by Jasantes** [comment]
Ok, thanks for your response, but the same issue perists, like all portable Chromium forks. But there is a solution, add to the executable shortcut: 
 --disable-machine-id --disable-encryption --user-data-dir="D:\...profile path"

**Comment by adeii** [comment]
Must be problem with portapp project. But --user-data-dir="../Data" could be added as flag as like as --no-sandbox.
Thanks for an idea.

**Comment by Jasantes** [comment]
BTW, I can't log into the profile account to syncronize. It looks greyed.
Should I open a new issue?

**Comment by Jasantes** [comment]
Perfect! Thank you very much and greetings from Uruguay.

**Comment by nima-amin** [comment]
It's marked closed but the problem persists.
Librewolf on startup creates a folder in C:Users ... which remains empty (fortunately) - the profiles are stored within the application directory.
Not so with Zen and Supermium. Zen stores the profiles entirely in C. Supermium (portableapps) has strange enough two user data folders, one in C, the other within the application directory.

**Comment by nima-amin** [comment]
Nope, folders in Local and Roaming again. Maybe I did it wrong. I deleted the two folders in Local and Roaming, then openend a new empty file in editor and saved it as zen-portable.bat in the zen programme folder (together with folders app, data, log and reg and the files portapp-prev.json, zen-portable.sample.yml, zen-portable-exe, zen-portable-win10.exe and portapp.json.
Would I have needed to write something in that bat file?

**Comment by adeii** [comment]
Add at the end of *profile.bat following:

timeout 11
rmdir %appdata%\snowfox /q /s
rmdir %localappdata%\snowfox /q

**Comment by nima-amin** [comment]
Nope - unless I did this bat creating wrong.
This is what I copied from your links in the editor and saved as zen-portable.bat:

@echo off
set APP_DIR=%~dp0\app
set DATA_DIR=%~dp0\data
"%APP_DIR%\zen.exe" -profile "%DATA_DIR%\profile1" -no-remote
timeout 11
rmdir %appdata%\zen /q /s
rmdir %localappdata%\zen /q

I had zen open definitely longer than 10 seconds.

**Comment by adeii** [comment]
Try increase timeout to 20 or 30.

**Comment by nima-amin** [comment]
@Snowfox:

Nope, language settings are saved now, but the portability problem still is not solved. Nothing is deleted, and voila the Snowfox-folders in Local and Roaming are full with files.

![Image](https://github.com/user-attachments/assets/5b8bc222-4d9c-4c57-ab72-fb57736244f5)

![Image](https://github.com/user-attachments/assets/4348dfe9-1e57-4c15-a23e-563afc0b6532)

**Comment by adeii** [comment]
So, this will demand more fixing inside *fox.exe or browser/omni.ja than just flags.
I will reconsider this later https://github.com/Heptazhou/Snowfox-Portable/blob/master/README.md

**Comment by adeii** [comment]
No (more) complainings, no issues.

**Comment by ioc2e3** [comment]
> According to project https://github.com/wysh3/Zen-Browser-Portable, its launcher\zenwindowsportable.bat would make Zen full portable for USB-flash.

Sorry , that ...

Here ( https://github.com/wysh3/Zen-Browser-Portable ) ...

It is now read-only ...

in addition ...

Here ( https://github.com/zen-browser/desktop ) and ( https://zen-browser.app/download/ ) ...

There is no Portable version ...

.
